api-reference/auth/desktop-sdks/web.mdx

```diff
@@ -178,26 +223,149 @@ const userInfo = connect({
 
 <Note>`web3.eth.getAccounts` will also initiate login automatically if user isn't already logged in.</Note>
 
+#### Log in with a Specific Social Platform
+
+You can configure the login flow to bypass the login modal entirely, prompting the user to log in directly with a specific social platform.
+
+The following example demonstrates how to prompt the user to log in with their Google account, skipping the login modal.
+
+```tsx App.tsx
+  // Handle user login
+  const handleLogin = async () => {
+    if (!connected) {
+      await connect({
+        socialType: "google",
+      });
+    }
+  };
+```
+
 ***
 
+### Verify the User's Login Status
+
+The `useConnect()` hook provides several functions to check the user's connection status:
+
+- `connected` — Returns a boolean value indicating whether the user is logged in (`true`) or not (`false`). This is useful for managing login logic.
+
+- `connectionStatus` — Returns a string representing the current stage of the user's flow:
+  - `loading`
+  - `connecting`
+  - `connected`
+  - `disconnected`
+
+  You can use `connectionStatus` to manage login logic or to display relevant status information in the UI.
```

Comment:
Do we want to add a code snippet here?

Reply:
Sure, maybe a useEffect monitoring connection status with these methods

Reply:
It seems like we don't need useEffect; `connectionStatus` handles everything automatically. I added a simple code snippet.